cloud_experts_tutorials/cloud-experts-rosa-with-hcp-private-offer-acceptance-and-sharing.adoc

```diff
@@ -0,0 +1,187 @@
++
+.Private offer selection dropdown
++
+image::rosa-private-offer-selection-dropdown.png[]
+
+. Figure 4 shows the bottom part of the offer page with the offer details. Check that the offer configuration reflects your expectations. 
++
+[NOTE]
+====
+The contract end date, the number of units included with the offer, and the payment schedule. In this example, 1 cluster and up to 3 nodes utilizing 4 vCPUs are included.
+====
++
+.Private offer details
++
+image::rosa-private-offer-details.png[]
+
+. Optionally, you can link:https://docs.aws.amazon.com/marketplace/latest/buyerguide/buyer-purchase-orders.html[add your own purchase order (PO) number] to the subscription that is being purchased, so it will be included on your subsequent AWS invoices. Also, check the _Additional usage fees_ that are charged for any usage above the scope of the _New offer configuration details_.
++
+[NOTE]
+====
+Private offers have several available configurations. It is possible that the private offer you are accepting is set up with a fixed future start date. If you do not have another active {hcp-title} subscription at the time of accepting the private offer, a public offer or an older private offer entitlement, accept the private offer itself and continue with the account linking and cluster deployment steps after the specified service start date. Neither of these steps is possible without an active {hcp-title} entitlement. Service start dates are always reported in the UTC time zone.
+====
+
+. If all the details match your expectations, click the *Create contract* button, shown in Figure 5, for private offers accepted by an AWS account that does not have the {hcp-title} activated yet and is creating the first contract for this service.
++
+.Create contract button
++
+image::rosa-create-contract-button.png[]
+
+. Or click the *Upgrade current contract button* for agreement-based offers as seen in Figures 4 and 6.
++
+.Upgrade contract button
++
+image::rosa-upgrade-contract-button.png[]
+
+. There is one more confirmation modal window shown, similar to the one shown in Figure 7.
++
+.Private offer acceptance confirmation window
++
+image::rosa-private-offer-acceptance-confirmation-window.png[]
+
+. If the accepted private offer service start date is set to be immediately following the offer acceptance, you can click the *Set up your account* button in the confirmation modal window shown in Figure 8.
++
+.Subscription confirmation
++
+image::rosa-subscription-contfirmation.png[]
+
+. If the accepted private offer has a future start date specified, return to the private offer page after the service start date, these are always in UTC time zone, and click the *Setup your account* button to proceed with the Red{nbsp}Hat and AWS account linking. With no agreement active, the account linking described below is not triggered, the _Account setup_ process can be done only after the _Service start date_.
```

Comment:
"If the accepted private offer has a future start date specified (start time will always be in UTC time zone), return to the private offer page after the service start date and click the Setup your account button to link your Red Hat and AWS accounts. With no agreement active, the account linking is not triggered and the account setup process can be done only after the Service start date."

Reply:
I'm getting ready to push an update with part of this step broken out into a note to make it easier to parse. Please let me know if that makes sense 😅